src/dosbox.cpp

```diff
@@ -797,6 +799,13 @@ void DOSBOX_Init()
 	        "you're using a copy-on-write or network-based filesystem, this setting avoids\n"
 	        "triggering write operations for these write-protected files.");
 
+	pbool = secprop->Add_bool("media_delay", when_idle, true);
+	pbool->Set_help(
+	        "Emulates media delay for mounted HDD, CDROM, and floppy drives.\n"
+	        "This is useful for games that have problems if media is too fast,\n"
+	        "      such as Deus (1996), Ishar 3 (1994), Robinson's Requiem (1994),\n"
+	        "      and Time Warriors (1997).");
```

Comment:
How about renaming this to `disk_access_delay` and using the below description?

```
Emulate disk access delays ('on' by default).
  on:   Emulate disk access delays for floppy drives, hard drives, and CD-ROM
        drives (default). This is preferable as many DOS games crash or
        misbehave if the disk access times are unrealistically fast (e.g.,
        Deus, Ishar 3, Robinson's Requiem, Time Warriors, Chasm - The Rift,
        Start Trek - A Final Unity, Emergency Room, etc.)
  off:  Disable disk access delay emulation. There is generally no benefit to
        this as disk access is fast enough even with the delay emulation
        enabled.
```

---

"Media" is such an overloaded term, and it's really isn't clear what it refers to. Especially in a "multimedia" app like DOSBox where "media" could mean audio or video as well.

The term "disk access delay" is still short and user-friendly and 100% unambigous.